heltour/tournament/static/tournament/js/edit_rosters.js

```diff
@@ -134,35 +134,23 @@ function updateAverage($teams) {
         var $players = $team.find('.player');
         var n = 0;
         var total = 0.0;
-        var nExp = 0;
-        var totalExp = 0.0;
         for (var j = 0; j < $players.length; j++) {
             var rating = parseInt($players.eq(j).attr('data-rating'));
             if (!isNaN(rating)) {
                 n += 1;
                 total += rating;
             }
-            var ratingExp = parseInt($players.eq(j).attr('data-exp-rating'));
-            if (!isNaN(ratingExp)) {
-                nExp += 1;
-                totalExp += ratingExp;
-            }
         }
         if (n > 0) {
             $team.find('.average-rating').text((total / n).toFixed(2));
         } else {
             $team.find('.average-rating').text('');
         }
-        if (nExp > 0) {
-            $team.find('.average-exp-rating').text((totalExp / nExp).toFixed(2));
-        } else {
-            $team.find('.average-exp-rating').text('');
-        }
     }
 }
 
 function updateDisplayedRatings() {
-    attrName = $('#id_rating_type').val() === 'expected' ? 'data-exp-rating' : 'data-rating';
+    attrName = $('#id_rating_type').val() === 'data-rating';
```

Comment:
This change is not correct. The previous one was checking to see if that rating type form element was set to 'expected'. If it was, it would set the `attrName` variable equal to `'data-exp-rating'. If not, it would set it to `'data-rating`

The new version sets attrName to `false`. becuase `id_rating_type` doesn't exist and  the value will be undefined which does not strictly equal 'data-rating'.